cmd/coder/resize_unix.go

```diff
@@ -0,0 +1,39 @@
+// +build !windows
+
+package main
```

Comment:
The resize files could potentially be put into the xterminal package since it makes more sense there.